### PR TITLE
Show task run retry count in pipeline diagram

### PR DESCRIPTION
**Pull request by evidolob**
Resolve: #357 

Example:
Screenshot 2021-07-02 at 10 05 02

### CODE REVIEW

**Comment by sudhirverma** [comment]
Screenshot 2021-07-07 at 12 05 12 PM

it doesn't display me retry count for task

**Comment by evidolob** [comment]
@sudhirverma  It will show only for PipelineRun

**Comment by sudhirverma** [comment]
> @sudhirverma It will show only for PipelineRun

I have open pipelineRun diagram only.

**Comment by sudhirverma** [comment]
new screenshot

Screenshot 2021-07-07 at 12 12 57 PM

File: test/pipeline/pipeline-graph.test.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ import * as graph from '../../src/pipeline/pipeline-graph';
 import { VirtualDocument } from '../../src/yaml-support/yaml-locator';
 import { PipelineRunData } from '../../src/tekton';
 import { tektonFSUri, tektonVfsProvider } from '../../src/util/tekton-vfs';
+import { NodeData } from '../../src/webview/pipeline-preview/model';
```
Comment (sudhirverma) [inline test/pipeline/pipeline-graph.test.ts:14]: can you remove this line.

**Comment by sudhirverma** [comment]
It's working fine for me I was having a problem with pipelineRun.

**Review by sudhirverma — APPROVED** [review]

**Pull request merged**